src/components/tables/Captures/index.tsx

```diff
@@ -1,115 +1,17 @@
-import {
-    Box,
-    Table,
-    TableBody,
-    TableCell,
-    TableContainer,
-    TableHead,
-    TableRow,
-    Typography,
-} from '@mui/material';
-import Error from 'components/shared/Error';
-import Empty from 'components/tables/Captures/Empty';
-import Rows from 'components/tables/Captures/Rows';
-import { FormattedMessage, useIntl } from 'react-intl';
-import { TABLES } from 'services/supabase';
-import { useQuery, useSelect } from 'supabase-swr';
-
-interface Discovers {
-    'capture_name': string;
-    'updated_at': string;
-    'job_status->>type': string;
-    'id': string;
-}
-
-const DISCOVERS_QUERY = `
-    capture_name, 
-    updated_at, 
-    job_status->>type, 
-    id
-`;
-
-const columns = ['data.name', 'data.status', 'data.updated_at'];
-
-const columnStyling = {
-    maxWidth: '20%',
-    textOverflow: 'ellipsis',
-    width: '20%',
-};
+import { Box } from '@mui/material';
+import EntityTable from 'components/tables/EntityTable';
```

Comment:
Using the entity table in here - it is more than good enough right now to have it be used on this page.